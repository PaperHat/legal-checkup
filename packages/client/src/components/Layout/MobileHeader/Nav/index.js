import styled from 'styled-components'

const Nav = styled.nav`
  background-color: #444141;
  display: inline;
  height: 35px;
  margin-left: -130px;
  margin-top: 50px;
  width: 100%;
`
Nav.displayName = 'Nav'

export default Nav
