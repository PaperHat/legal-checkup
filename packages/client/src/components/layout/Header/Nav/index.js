import styled from 'styled-components'
import styledMap from 'styled-map'

import { DESKTOP, MOBILE, TABLET, GREYISH_BROWN } from '../../../../constants'

const Nav = styled.nav`
  display: flex;

  ${styledMap('format', {
    [`${DESKTOP}`]: `
      width: 100%;
      margin-left: 100px;
    `,
    [`${TABLET}`]: `
      margin-left: 100px;
    `,
    [`${MOBILE}`]: `
      padding: 0 3px;
      height: 43px;
      background-color: ${GREYISH_BROWN};
      align-items: flex-end;
    `
  })};
`
Nav.displayName = 'StyledNav'

export default Nav
