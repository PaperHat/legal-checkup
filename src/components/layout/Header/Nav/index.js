import styled from 'styled-components'
import styledMap from 'styled-map'

import { DESKTOP, MOBILE, TABLET, BACKGROUND_NAV } from '../../../../constants'

const Nav = styled.nav`
  display: flex;

  ${styledMap('format', {
    [`${DESKTOP}`]: `
      width: 100%;
      margin-left: 100px;
    `,
    [`${TABLET}`]: `
      margin-left: 100px;
    `,
    [`${MOBILE}`]: `
      padding: 0 3px;
      height: 43px;
      background-color: ${BACKGROUND_NAV};
      align-items: flex-end;
    `
  })};
`
Nav.displayName = 'StyledNav'

export default Nav
