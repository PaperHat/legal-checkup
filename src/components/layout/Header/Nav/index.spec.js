import * as React from 'react'
import { create } from 'react-test-renderer'

import Nav from './'

describe('components:layout:Header:Nav', () => {
  it('matches the snapshot', () => {
    expect(create(<Nav>Nav</Nav>).toJSON()).toMatchSnapshot()
  })
})
