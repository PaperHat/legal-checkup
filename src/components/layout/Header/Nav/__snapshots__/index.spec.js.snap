// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:layout:Header:Nav matches the snapshot 1`] = `
<nav
  className="sc-bdVaJa bNoxQU"
>
  Nav
</nav>
`;
